videointelligence/google/cloud/videointelligence_v1/gapic/transports/video_intelligence_service_grpc_transport.py

```diff
@@ -71,7 +72,8 @@ def __init__(
         self._stubs = {
             "video_intelligence_service_stub": video_intelligence_pb2_grpc.VideoIntelligenceServiceStub(
                 channel
-            )
+            ),
+            "operations_stub": operations_pb2.OperationsStub(channel)
```

Comment:
Even if we were making manual changes, this stub is unused (the `_operations_client` created by the constructor manages its own stub).

Reply:
Removed.